tests/test_abspath.py

```diff
@@ -278,6 +279,40 @@ def test_abspath_read(local_v6_txt):
     assert u.read(byte=True) == v6_txt_contents().encode()
 
 
+def test_abspath_find_all_files_and_rmdir(local_test_path):
+    """Test two methods:
+        - find_all_files()
+        - rmdir()
+
+    Make a directory structure with empty files
+    (and an empty directory for local storage only).
+
+    Check if find_all_files() returns correct file (not sub-directory) paths.
+    Check if rmdir() deletes all empty files on given $prefix.
+
+    For local storage, check if find_all_files() does not return
+    an empty dir.
+    """
+    prefix = os.path.join(local_test_path, 'test_abspath_find_all_files_and_rmdir')
+    all_files = make_files_in_dir(prefix, make_local_empty_dir_d_a=True)
+
+    # test find_all_files()
+    all_files_found = AbsPath(prefix).find_all_files()
+    assert sorted(all_files_found) == sorted(all_files)
+    for file in all_files:
+        assert AbsPath(file).exists
+
+    # test rmdir(dry_run=True)
+    AutoURI(prefix).rmdir(dry_run=True)
+    for file in all_files:
+        assert AbsPath(file).exists
+
+    # test rmdir(dry_run=False)
+    AutoURI(prefix).rmdir(dry_run=False)
+    for file in all_files:
+        assert not AbsPath(file).exists
```

Comment:
Would be good to see this separated into three tests. Could use a shared fixture to create the test files only once.

Reply:
Separated them into two tests (one for each method). Will think about making them as fixtures in the next refactoring since all other tests are based on non-fixture dir/files.